### PR TITLE
SEB leads debugger - lead creation

### PR DESCRIPTION
**Pull request by lucas-larsson**
## Describe your changes

Adds SEB leads debugger that allows to create lead entries in Insurely service manually

First version returns ID of created entry without processing it further

Since this is test-only thing, disabled it in production environment

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/app/debugger/seb-leads/createSebLead.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    const errorMessage = error instanceof Error ? error.message : 'Unknown error'
+
+    return {
+      errors: {
+        generic: [errorMessage],
+      },
+    }
+  }
+
+  const message = `Lead created, sebSessionId=${sebSessionId}`
```
Comment (Youakeem) [inline apps/store/src/app/debugger/seb-leads/createSebLead.tsx:46]: Nit: What do you think about moving this logic inside the `try` block?

File: apps/store/src/app/debugger/seb-leads/createSebLead.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  console.debug(`Creating SEB lead for ${params.personalNumber}`, body)
+
+  const response = await fetch(url, {
+    method: 'POST',
+    body: body,
+    headers: headers,
+  })
+
+  if (!response.ok) {
+    const json = (await response.json()) as CreateSebLeadsResponse
```
Comment (Youakeem) [inline apps/store/src/app/debugger/seb-leads/createSebLead.tsx:98]: Seems like we are using `await response.json()` twice. Can't we get the json right after the response and just throw if it's not ok?

**Review by Youakeem — APPROVED** [review]
Great work, man! I left a couple of small comments!

**Comment by alebedev** [comment]
### Merge activity

* **Sep 5, 7:14 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4650).

**Pull request merged**